api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- <databaseChangeLog
+		xmlns="http://www.liquibase.org/xml/ns/dbchangelog/2.0"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/2.0
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd"> -->
+
+<databaseChangeLog logicalFilePath="liquibase-update-to-latest.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd
+    http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+
+	<changeSet id="1.0" author="Partners In Health">
+		<preConditions onError="MARK_RAN" onFail="MARK_RAN">
+			<not><tableExists tableName="idgen_identifier_source" /></not>
```

Comment:
Each table creation should have its own changeset.

Reply:
okay